backend/apps/progress/views.py

```
@@ -811,6 +811,31 @@ class CertificateVerificationThrottle(AnonRateThrottle):
     rate = "10/minute"
 
 
+class CertificateVerifyView(APIView):
+    """Public API to verify certificate by hash."""
+    
+    permission_classes = []  # Public endpoint
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)